packages/web/src/components/molecules/SupplySummaly/index.tsx

```diff
@@ -69,15 +69,15 @@ export const SupplySummary = ({ apy, creators, annualSupplyGrowthRatio }: Props)
       <SupplySummaryContainer>
         <SupplyBadge>
           <Statistics>{apy?.dp(2).toNumber()}%</Statistics>
-          <span>APY for Stakers</span>
+          <span>APY Stakers</span>
         </SupplyBadge>
         <SupplyBadge>
           <Statistics>{creators?.dp(2).toNumber()}%</Statistics>
-          <span>APY for Creators</span>
+          <span>APY Creators</span>
         </SupplyBadge>
         <SupplyBadge>
           <Statistics>{annualSupplyGrowthRatio?.dp(2).toNumber()}%</Statistics>
-          <span>ASG of DEV</span>
+          <span>ASG DEV</span>
```

Comment:
https://www.figma.com/file/2XztNCtx3iXyPECgnNtIAZ/Stakes-Social.-0.1?node-id=360%3A0
It differs from the wording in Figma.
Is this OK?

see also: https://github.com/dev-protocol/stakes.social/pull/577

Reply:
APY Stakers is fine @hhatto